.github/workflows/CI.yaml

```diff
@@ -22,8 +22,17 @@ on:
     branches:
       - master
   pull_request:
-#  schedule: TODO: unpin minor lib versions and check weekly
-#    - 0 0 0 ? * FRI *
+    paths-ignore:
+      - "*.md"
+      - "*.txt"
+      - ".asf.yaml"
+      - ".dlc.json"
+      - ".licenserc.yaml"
+      - "docs/*"
+      - ".github/workflows/*"
```

Comment:
Pull requests require the CI check status context named `CheckStatus` to pass before they can be merged. And the check context is generated by the job `CheckStatus` in this file (line 123), if changes are in these `paths-ignore`, the workflow won't be run and the check status context won't be reported, thus all PRs can't be merged, you need to either remove these paths-ignore, or add a dummy workflow in a separate file that run a empty job named `CheckStatus` to generate the required context, but it is hard to set the reversed condition of these `paths-ignore`, you can see how I set in the main repo but that is still not perfect because under some cases the 2 workflows are all executed

Reply:
Ah I see how it works, gonna fix it later.

Reply:
> Pull requests require the CI check status context named `CheckStatus` to pass before they can be merged. And the check context is generated by the job `CheckStatus` in this file (line 123), if changes are in these `paths-ignore`, the workflow won't be run and the check status context won't be reported, thus all PRs can't be merged, you need to either remove these paths-ignore, or add a dummy workflow in a separate file that run a empty job named `CheckStatus` to generate the required context, but it is hard to set the reversed condition of these `paths-ignore`, you can see how I set in the main repo but that is still not perfect because under some cases the 2 workflows are all executed

Looks like the paths_ignore option doesn't work like what I thought.. it's crazy. 

Maybe the diff method is the ultimate solution given third party action cannot be run in asf repos. 

Diff method:
https://github.community/t/paths-ignore-not-working-for-base-level-files/18445/4

Reply:
Some third-party GHAs are approved by Apache infra and can be used, @Superskyyy take a look at [this](https://github.com/apache/skywalking-cli/blob/60cee4a926a867a7bd934b7f94c9a8517e141608/.github/workflows/go.yml#L34), though I haven't tried to use this GHA to achieve my goal as stated above

Reply:
I'm gonna try it and let you know if it works.